packages/api/tests/lambdas/test-bulk-operation.js

```diff
@@ -689,33 +697,44 @@ test.serial('bulk operation BULK_GRANULE_DELETE does not throw error for granule
 
 test.serial('bulk operation BULK_GRANULE_REINGEST reingests list of granule IDs', async (t) => {
   await setUpExistingDatabaseRecords(t);
-  const granules = t.context.granules;
+  const { granules, knex } = t.context;
+  const payload = {
+    ids: [
+      granules[0].granuleId,
+      granules[1].granuleId,
+    ],
+  };
 
   await bulkOperation.handler({
     type: 'BULK_GRANULE_REINGEST',
     envVars,
-    payload: {
-      ids: [
-        granules[0].granuleId,
-        granules[1].granuleId,
-      ],
-    },
+    payload,
     reingestHandler: reingestStub,
   });
 
   t.is(reingestStub.callCount, 2);
-  reingestStub.args.forEach((callArgs) => {
+  reingestStub.args.forEach(async (callArgs) => {
     const matchingGranule = granules.find((granule) =>
       granule.granuleId === callArgs[0].granule.granuleId);
 
-    t.deepEqual(matchingGranule, callArgs[0].granule);
+    const pgGranule = await getUniqueGranuleByGranuleId(knex, matchingGranule.granuleId);
+    const translatedGranule = await translatePostgresGranuleToApiGranule({
+      granulePgRecord: pgGranule,
```

Comment:
```suggestion
    const translatedGranule = await translatePostgresGranuleToApiGranule({
      granulePgRecord: matchingGranule,
```

I think getting the unique granule query is unnecessary since we already have the single `matchingGranule`

Reply:
It's still necessary since `translatePostgresGranuletoApiGranule` requires a PG record to translate

Reply:
I see. It seems like both `getUniqueGranuleByGranuleId` and `translatePostgresGranuletoApiGranule` are unnecessary if you compare `matchingGranule` directly to `callArgs[0].granule`. Then you have to omit `dataType`, `version`, and `execution` though. So this is better. I'll resolve this comment